### PR TITLE
Update vpa

### PR DESCRIPTION
**Pull request by njuettner**
Taking the latest scrape from upstream

### CODE REVIEW

**Review by TheoBrigitte — CHANGES_REQUESTED** [review]
Please update the `Chart.yaml` so we know which upstream version we are at.

I would have prefer using `git subtree` to handle upstream sync, but I'm fine with this.
We have some good doc about this https://github.com/giantswarm/template-app/blob/master/CONTRIBUTING.md

**Comment by njuettner** [comment]
By upstream version you the appVersion or Version? The appVersion so far hasn't changed https://github.com/FairwindsOps/charts/blob/master/stable/vpa/Chart.yaml#L5-L6

Ok I didn't know thanks ❤️. Next time 🙂

**Comment by TheoBrigitte** [comment]
Oh so you took latest changes from master which are some kind of 0.9.2-dev version ?
What I am interested in is to know which tag/commit we are synced against. If we synced against latest master, lets put the commit into `appVersion`.

File: helm/vertical-pod-autoscaler-app/Chart.yaml [hunk 1]
```diff
@@ -11,5 +11,7 @@ sources:
 type: application
 version: [[ .Version ]]
 annotations:
+  repository: https://github.com/FairwindsOps/charts/tree/master/stable/vpa
```
Comment (njuettner) [inline helm/vertical-pod-autoscaler-app/Chart.yaml:14]: As we agreed, adding the upstream repo and the used commit make sense

File: helm/vertical-pod-autoscaler-app/values.yaml [hunk 2]
```diff
@@ -101,7 +109,7 @@ updater:
 
 admissionController:
   # admissionController.enabled -- If true, will install the admission-controller component of vpa
-  enabled: false
+  enabled: true
```
Comment (njuettner) [inline helm/vertical-pod-autoscaler-app/values.yaml:112]: We enable admission controller by default, we can remove the setting in config repo afterwards

File: helm/vertical-pod-autoscaler-app/values.yaml [hunk 3]
```diff
@@ -116,7 +124,12 @@ admissionController:
     env: {}
   # admissionController.cleanupOnDelete -- If true, a post-delete job will remove the mutatingwebhookconfiguration and the tls secret for the admission controller
   cleanupOnDelete: true
-  replicaCount: 1
+  # admissionController.replicaCount -- The amount admission-controller pods which should run.
+  replicaCount: 2
+  # admissionController.podDisruptionBudget -- This is the setting for the pod disruption budget
+  podDisruptionBudget:
+    # maxUnavailable: 1
+    maxAvailable: 1
```
Comment (njuettner) [inline helm/vertical-pod-autoscaler-app/values.yaml:132]: PDB is integrated now and we only terminate one pod at a time

File: helm/vertical-pod-autoscaler-app/values.yaml [hunk 4]
```diff
@@ -68,10 +71,15 @@ updater:
   # updater.enabled -- If true, the updater component will be deployed
   enabled: true
   # updater.extraArgs -- A key-value map of flags to pass to the updater
-  extraArgs: {}
+  extraArgs:
+    # update.extraArgs.min-replicas -- Minimum number of replicas to perform update
+    min-replicas: 1
+    # update.extraArgs.v -- Verbosity level for updater
+    v: 2
```
Comment (njuettner) [inline helm/vertical-pod-autoscaler-app/values.yaml:78]: This is the setting which we extract from config repo, IMHO it makes sense. We can drop the config repo setting afterwards.

**Comment by njuettner** [comment]
@TheoBrigitte PTAAL 🙏🏻

File: helm/vertical-pod-autoscaler-app/Chart.yaml (hunk 1 above)
Comment (TheoBrigitte) [reply helm/vertical-pod-autoscaler-app/Chart.yaml:14]: <3 Thanks

File: helm/vertical-pod-autoscaler-app/values.yaml (hunk 2 above)
Comment (TheoBrigitte) [reply helm/vertical-pod-autoscaler-app/values.yaml:112]: PR https://github.com/giantswarm/config/pull/771

**Review by TheoBrigitte — APPROVED** [review]
Perfect :ok_hand:

**Pull request closed without merging**